Automattic/newspack-plugin
# Reorganise reader revenue

### All Submissions:






### Changes proposed in this Pull Request:

This PR reorganises the Reader Revenue, displaying the platform screen first and then using `Grid` throughout the various screens

### How to test the changes in this Pull Request:

1. Check the Reader Revenue -- Test all the different platform options and menu items
2. Switch to this branch
3. Repeat 1.

### Other information:
